Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -620,10 +621,46 @@ ShaderCode GeneratePixelShaderCode(APIType api_type, const ShaderHostConfig& hos
 
   const bool use_dual_source = host_config.backend_dual_source_blend;
   const bool use_shader_blend = !use_dual_source && host_config.backend_shader_framebuffer_fetch;
+  const bool use_shader_logic_op =
+#ifdef __APPLE__
+      !host_config.backend_logic_op && host_config.backend_shader_framebuffer_fetch;
+#else
+      false;
```

Comment:
A separate PR to add shader logic ops support on OpenGL ES (Android) will be opened after this is merged.